### PR TITLE
feat: create jira tickets.

### PR DESCRIPTION
**Pull request by seilagamo**
Add a button to create tickets in Jira.
Also, create a modal to modify the summary and description fields that will be sent to the API.
Avoid showing the Jira integration to teams that are not onboarded in Vulcan Tracker.

### CODE REVIEW

File: src/report/findingview/findingview.vue [hunk 1]
```diff
@@ -58,8 +60,10 @@ import {
 export default class FindingView extends Vue {
   private apiUrl: string = "";
   private teamId: string = "";
+  private teamOnBoardedVT: boolean = false;
   private findingId: string = "";
   private findingsApi!: FindingsApi;
+  private teamsApi?: TeamsApi;
```
[the remark this answers is not in the document]
Comment (seilagamo) [reply src/report/findingview/findingview.vue:66]: I can put this as not nullable but the linter complains about it

[the remark this answers is not in the document]
Comment (seilagamo) [reply src/report/findingview/findingview.vue]: The alternative is to change the declaration with `private teamsApi: TeamsApi | undefined;` WDYT?

[the remark this answers is not in the document]
Comment (seilagamo) [reply src/report/findingview/findingview.vue]: Fixed in commit d326258948a8e4c539af9737a87716ad0d7cef9b

**Review by manelmontilla — APPROVED** [review]
LGTM

**Pull request merged**